t/source.test

```diff
@@ -0,0 +1,3 @@
+--echo first line
+--source include/hello.inc
```

Comment:
Should we also test multi-level `--source`?
I.e.:
t/source.test:
--echo first line
--source include/hello.inc
--echo last line

include/hello.inc:
--echo Hello
--source include/world.inc
--echo !

include/world.inc:
--echo World

Reply:
I don't think that's needed